src/app/static/src/tests/components/modelCalibrate/calibrationResults.test.ts

```diff
@@ -0,0 +1,164 @@
+import registerTranslations from "../../../app/store/translations/registerTranslations";
+import {emptyState} from "../../../app/root";
+import {inactiveFeatures} from "../../../app/main";
+import {BarChartWithFilters} from "@reside-ic/vue-charts";
+import {modelCalibrate, ModelCalibrateState} from "../../../app/store/modelCalibrate/modelCalibrate";
+import {expectTranslated} from "../../testHelpers";
+import {BarchartIndicator} from "../../../app/types";
+import Vue from 'vue';
+
+const localVue = createLocalVue();
+
+const defaultSelections = {
+    indicator_id: "TestIndicator",
+    x_axis_id: "spectrum_region",
+    disaggregate_by_id: "type",
+    selected_filter_options: {
+        region: {id: "r1", label: "region 1"},
+        age: {id: "a1", label: "0-4"}
+    }
+}
+
+function getStore(modelCalibrateState: Partial<ModelCalibrateState> = {}, partialGetters = {}, partialSelections = {}) {
+    const store = new Vuex.Store({
+        state: emptyState(),
+        modules: {
+            modelCalibrate: {
+                state: {
+                    calibratePlotResult: mockCalibrateResultResponse({data: ["TEST DATA"] as any}),
+                    ...modelCalibrateState
+                },
+                namespaced: true,
+                getters: {
+                    indicators: jest.fn().mockReturnValue(["TEST BARCHART INDICATORS"]),
+                    filters: jest.fn().mockReturnValue(["TEST BAR FILTERS"]),
+                    ...partialGetters
+                },
+                mutations: modelCalibrateMutations
+            },
+            plottingSelections: {
+                namespaced: true,
+                state: {
+                    calibratePlot: defaultSelections,
+                    ...partialSelections
+                },
+                getters: {
+                    calibratePlotDefaultSelections: jest.fn().mockReturnValue(defaultSelections),
+                    ...partialGetters
+                },
+                mutations: plottingSelectionMutations
+            }
+        }
+    });
+    registerTranslations(store);
+    return store;
+}
+
+declare let currentUser: string;
+currentUser = "guest";
+
+describe("CalibrateResults component", () => {
+
+    it("renders title and text", () => {
+        const store = getStore();
+        const wrapper = shallowMount(CalibrationResults, {localVue, store});
+        const title = wrapper.find("h3");
+        // expectTranslated(title, "Review calibration results", "Examiner les résultats de l'étalonnage", store);
+        // const text = wrapper.find("p");
+        // expectTranslated(text, "Comparison of unadjusted estimates (uncalibrated) to model estimates calibrated to spectrum results as specified in calibration options above.", "Comparaison des estimations non ajustées (non calibrées) aux estimations du modèle calées sur les résultats du spectre comme spécifié dans les options de calage ci-dessus.", store);
```

Comment:
I cannot get this test to recognise the French text, it just receives English. It works in properly in the browser and I've set it up in the same way as the tests for `ModelOutputs` so I'm not sure what is going on.

Reply:
It fails on my machine. I haven't tried on Buildkite.

Reply:
Please try - I seem to have the same problem with this and other tests that work on Buildkite. If it works on Buildkite then I can give you a fix to try on your machine.

Reply:
Actually, on my machine it's sufficient to add a single `await Vue.nextTick();` after the `shallowMount` (similar to the following test) - although I don't know why this works. Please try that on your machine.